addons/account/models/account_bank_statement.py

```diff
@@ -58,7 +58,7 @@ def default_get(self, fields_list):
                 .filtered(lambda line: not line.statement_complete) \
                 .sorted()
             if not lines:
-                raise UserError(_('No editable line selected.'))
+                raise UserError(_('One or more selected lines already belong to a complete statement.'))
```

Comment:
This error message is now even more misleading.
It should be *All selected lines are already connected to a complete statement*

Reply:
Benj gave me this line, I didn't look much more in the context of it (should have). Will you correct it in one of your branches?

Reply:
Well it isn't wrong :sweat_smile: 
"One or more" includes "all" :upside_down_face:

Reply:
> This error message is now even more misleading.

I don't agree though. The previous comment really didn't mean anything unless you wrote the module. This one is just not accurate, but understandable for a regular user.

Reply:
@william-andre  You're right, somehow, but the previous error was not giving any information at all, so it was less misleading :D 
I had actually changed it, but it came back through one of the many revert/rebase/patches.
@vin-odoo Sure, will do :)